go.mod

```diff
@@ -1,27 +1,27 @@
 module sigs.k8s.io/cli-utils
 
-go 1.18
+go 1.20
 
 require (
 	github.com/google/go-cmp v0.5.9
 	github.com/google/uuid v1.3.0
-	github.com/onsi/ginkgo/v2 v2.7.0
-	github.com/onsi/gomega v1.24.2
-	github.com/spf13/cobra v1.6.1
+	github.com/onsi/ginkgo/v2 v2.9.5
+	github.com/onsi/gomega v1.27.7
+	github.com/spf13/cobra v1.7.0
 	github.com/spyzhov/ajson v0.7.2
-	github.com/stretchr/testify v1.8.1
+	github.com/stretchr/testify v1.8.2
 	gopkg.in/yaml.v3 v3.0.1
-	k8s.io/api v0.26.0
-	k8s.io/apiextensions-apiserver v0.26.0
-	k8s.io/apimachinery v0.26.0
-	k8s.io/cli-runtime v0.26.0
-	k8s.io/client-go v0.26.0
-	k8s.io/component-base v0.26.0
-	k8s.io/klog/v2 v2.80.1
-	k8s.io/kubectl v0.26.0
-	k8s.io/utils v0.0.0-20230115233650-391b47cb4029
-	sigs.k8s.io/controller-runtime v0.14.1
-	sigs.k8s.io/kustomize/kyaml v0.13.9
+	k8s.io/api v0.28.0-alpha.4
```

Comment:
Now, i just use  v0.28.0-alpha.4 to fix it, and i think we need to waiting for a stable version.